app/javascript/styles/mastodon/components.scss

```
@@ -2204,10 +2204,37 @@ a.account__display-name {
 
   .flex-spacer {
     background: transparent;
+    flex: 1 1 auto;
+  }
+
+  .compose-form {
+    flex: 1;
+    overflow-y: hidden;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .autosuggest-input {
+    padding-bottom: 11px;
+  }
+
+  .emoji-picker-dropdown {
+    position: relative;
+    top: 5px;
+    left: 234px;
+    z-index: 1;
+    width: 0;
+    height: 0;
+  }
+
+  .compose-form__autosuggest-wrapper {
+    overflow-y: auto;
+    background-color: white;
```

```suggestion
    background-color: $white;
```

I think this change will pass the test.
ref: https://codeclimate.com/github/tootsuite/mastodon/pull/10841